code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1137,10 +1137,15 @@ func (srh *StorageRestHandler) getUserStakePoolStat(w http.ResponseWriter, r *ht
 	var ups = new(userPoolStat)
 	ups.Pools = make(map[datastore.Key][]*delegatePoolStat)
 	for _, pool := range pools {
+		poolStatusStr, err := spenum.PoolStatus(pool.Status).String()
+		if err != nil {
+			common.Respond(w, r, nil, err)
```

Comment:
We don't have to response error when the pool has invalid status and ignore all other info, instead we should return all to users, and let users to determine what they should do base on the status.